modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -82,6 +82,70 @@ image:configuration/files/federation-3.png[image]
 +
 You may disconnect the share at any time by clicking the btn:[trash can] icon.
 
+== Federated Sharing Scanner CronJob Configuration
+
+[IMPORTANT]
+====
+As migration step to 10.5, before enabling this cronjob please remember to remove existing system cron that executes legacy **occ incoming-shares:poll**
+====
+
+Fed Shares Scanner is a background job used to run the federates shares
+scanner over federates shares that are eligible for scanning,
+to ensure integrity of the file cache. Scanner will search for federated shares
+that satisfy the below requirements:
+
+. ensure that within single cron run, at max [cronjob_scan_external_batch]
+scans will be performed out of all accepted external shares
+. scan of that external share has not been performed within
+last [cronjob_scan_external_min_scan] seconds
+. user still exists, and has been active recently, meaning logged in at
+least [cronjob_scan_external_min_login] seconds ago
+. federated share root etag/mtime changed, signaling that remote changed
+and requires scan
+
+To enable cronjob, go to Admin Settings: Federated Cloud Sharing and enable the checkbox
+
+image:configuration/files/sharing/federated-cloud-sharing/scan-cronjob.png[Checkbox]
+
+Alternatively you can use command line
+
++
+----
+occ config:app:set files_sharing cronjob_scan_external_enabled --value 'yes'
+----
+
+You can also configure specific settings of the cronjob:
+
+. To customize minimum amount of hours when last login of user happened so scan is triggered (ensures only active users get fed shares synced)
```

Comment:
```suggestion
. To customize the minimum amount of time since last login of user so scan is triggered (ensures only active users get fed shares synced)
```